modules/View.js

```diff
@@ -93,30 +159,34 @@
 
     /**
      * Hotswap rendering system reroute method.
+     * triggers templateRenderTiming on view to track render times
      * @method templateRender
      * See Torso.templateRenderer#render for params
      */
     templateRender: function(el, template, context, opts) {
       // Detach just this view's child views for a more effective hotswap.
       // The child views will be reattached by the render method.
+      var pageName = null;
+      if(opts){
+        pageName = opts.pageName;
+      }
+
+      var uuid = (new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+      this.trigger('templateRenderTiming', {
```

Comment:
same here. Name should be something like: 'pre-render' and 'post-render' and the payload doesn't need state. That should be inferred by which event you are listening to.